cmd/contour/serve.go

```diff
@@ -501,7 +525,10 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 	g.Add(lbsw.Start)
 
 	// Register an informer to watch envoy's service if we haven't been given static details.
-	if ctx.IngressStatusAddress == "" {
+	if ctx.IngressStatusAddress != "" {
+		log.WithField("loadbalancer-address", ctx.IngressStatusAddress).Info("Using supplied information for Ingress status")
```

Comment:
s/loadbalancer-address/ingress-status-address to keep consistent with the config name.

Reply:
This field was present in the original code (the log line must moved). We should keep this field name because it correlates with the log message in `serve.go`.

Reply:
I don't agree, Contour should be consistent with its logs and matching the same name against what it's logging is what we should strive to do.

Reply:
This is consistent. You need to think about logs from the perspective of someone trying to triage a problem in production. That person doesn't know what the internal names are in the code, but can correlate context tags if we use them consistently.

When we use different tag names, we make life harder for operators. If we apply your suggestion, then there are different context tags for setting the address in config vs. applying the address to an object. That makes is unnecessarily hard for operators to understand what Contour is doing with the address.

At any rate, this code has not changed in this PR.

This PR:
```
cmd/contour/ingressstatus.go:                   isw.log.WithField("loadbalancer-address", lbAddress(lbs)).
cmd/contour/serve.go:           log.WithField("loadbalancer-address", ctx.IngressStatusAddress).Info("Using supplied information for Ingress status")
```

main:
```
cmd/contour/ingressstatus.go:                   isw.log.WithField("loadbalancer-address", lbAddress(lbs)).Info("received a new address for status.loadBalancer")
cmd/contour/serve.go:           log.WithField("loadbalancer-address", ctx.IngressStatusAddress).Info("Using supplied information for Ingress status")
```

Reply:
Right, to be consistent we should use the name that operators use to configure the address. When specifying an address they set `ingress-status-address`, not `loadbalancer-address` which is what is log is referencing.

Reply:
If you don't want to change the the log line @jpeach I'll do a follow up PR to make it consistent.

Reply:
This patch doesn't change the existing code. I think that if we want to make that change, it would be best to have a separate issue.